packages/cli/lib/errorHandlers/overrrideErrors.js

```diff
@@ -0,0 +1,95 @@
+const { isSpecifiedError } = require('@hubspot/local-dev-lib/errors/apiErrors');
+const { logger } = require('@hubspot/local-dev-lib/logger');
+
+const { PLATFORM_VERSION_ERROR_TYPES } = require('../constants');
+const { i18n } = require('../lang');
+const { uiLink } = require('../ui');
+
+const i18nKey = 'cli.lib.errorHandlers.overrideErrors';
+
+function createDashes() {
+  return '-'.repeat(79);
+}
+
+function createPlatformVersionError(subCategory, errorData) {
+  const platformVersion = errorData.context.PLATFORM_VERSION[0];
+  const docsLink = uiLink(
+    i18n(`${i18nKey}.platformVersionErrors.docsLink`),
+    'https://developers.hubspot.com/docs/platform/platform-versioning'
+  );
+  let translationKey;
+
+  if (
```

Comment:
this could be a bit cleaner as an object that maps subcategory to i18nkey, i.e.
```
{
  [PLATFORM_VERSION_ERROR_TYPES.PLATFORM_VERSION_RETIRED]: 'platformVersionRetired'
}

Reply:
Great idea! Addressed in https://github.com/HubSpot/hubspot-cli/pull/1016/commits/93170ea1751b72db2582750d67c21b76b20d4766